msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -1524,6 +1528,131 @@ The following code provides an example of meeting end event payload:
 | **value.EndTime** | The meeting end time in UTC. |
 | **locale**| The locale of the message set by the client. |
 
+### Receive meeting participant events
+
+Your bot can receive real-time meeting events such as participant join and leave events through the meeting event subscription in Developer Portal. A bot receives the participant events only if subscribed to these events in Developer Portal.
+
+> [!NOTE]
+>
+> * Participant events are supported only for scheduled meetings.
+> * For a bot to receive participant events, ensure that you add the bot to the meeting before a participant joins or leaves the meeting.
+
+To subscribe to participant events, follow these steps:
+
+1. In [Developer Portal](https://dev.teams.microsoft.com/) open your bot app or import an existing app.
+1. In the **Meeting event subscriptions** section, select the events:
+    * Participant join
+    * Participant leave
+1. Select **Save**
+
+   :::image type="content" source="~/assets/images/apps-in-meetings/participant-events.png" alt-text="Screenshot shows how developer portal display for participant events.":::
```

Comment:
Can you double check this image is available? When view it in the markdown view, the image doesn't render.

Reply:
Image is rendering on the review site. Here is snippet for the quick reference.

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/91744224/1ff3dc2d-5b19-4727-8019-bbb8b4ab5f82)